conans/model/options.py

```diff
@@ -188,6 +191,12 @@ def __setitem__(self, item, value):
     def pop(self, item):
         return self._reqs_options.pop(item, None)
 
+    def remove(self, name, package):
```

Comment:
Maybe a bit confusing this interface. Maybe a ``remove_package``, and ``remove_option`` methods?